pyrovision/models/pyronear_model.py

```diff
@@ -0,0 +1,34 @@
+ # -*- coding: utf-8 -*-
+
+# Copyright (c) Pyronear contributors.
+# This file is dual licensed under the terms of the CeCILL-2.1 and AGPLv3 licenses.
+# See the LICENSE file in the root of this repository for complete details.
+
+import torch
+from pyrovision.models.utils import cnn_model
+import holocron
+from holocron.models.utils import load_pretrained_params
+
+
+## Define Model
+backbone = 'rexnet1_0x'
+num_classes = 1
+nb_features = 1280
+cut = -2
```

Comment:
All of those should be moved to default values of the function's argument :ok_hand:

Reply:
ok 👍